content/rest/reference/rate-limit.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Rate limit
+intro: 'With Rate Limit API you can check the current rate limit status of various REST API's.'
```

Comment:
```suggestion
intro: 'With the Rate Limit API, you can check the current rate limit status of various REST APIs.'
```
It looks like the `'` was causing issues in the YAML, so I am editing it out. This should fix the failing tests.